recipes/level-zero/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import get, apply_conandata_patches, copy, export_conandata_patches, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+import os
+
+
+class LevelZeroConan(ConanFile):
+    name = "level-zero"
+    license = "MIT"
+    homepage = "https://github.com/oneapi-src/level-zero"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "OneAPI Level Zero Specification Headers and Loader"
+    topics = ("api-headers", "loader", "level-zero", "oneapi")
+    package_type = "shared-library"
+
+    # Binary configuration
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": "10",
+            "clang": "7",
+            "gcc": "7",
+            "msvc": "190",
+            "Visual Studio": "15",
+        }
+
+    def requirements(self):
+        self.requires("spdlog/1.14.1")
+
+    def source(self):
+        version_data = self.conan_data["sources"][self.version]
+        get(self, **version_data, strip_root=True)
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def generate(self):
+        env = VirtualBuildEnv(self)
+        env.generate()
+        apply_conandata_patches(self)
```

Comment:
```suggestion
        apply_conandata_patches(self)
```

- VirtualBuildEnv is not needed as there is no build requirement. Need to remove the import too, otherwise the linter will complain about
- apply_conandata_patches should be move to build(), so will touch only the build folder: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L122 (do not need to add _patch_sources method, only move to build())